packages/frontend-core/src/fetch/DataFetch.js

```diff
@@ -62,6 +62,7 @@ export default class DataFetch {
       cursors: [],
       resetKey: Math.random(),
       error: null,
+      tenantOwner: null,
```

Comment:
I'd prefer not to add specific metadata like tenant owner into the generic data fetch - we should try and keep this fully generic. I also don't think it belong at this level. I think that if you need to know the tenant owner then you should do the required API call to get that data, rather than always fetching the tenant owner every time you do an API call to get users.